pom.xml

```diff
@@ -2344,8 +2378,25 @@
                 </plugin>
 
                 <plugin>
+
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-enforcer-plugin</artifactId>
+                    <version>3.3.0</version>
+                    <dependencies>
```

Comment:
to prevent issues with different Java versions. Java 8 was trying to load JDK 11-specific classes

Reply:
Where were the the JDK 11 specific classes being loaded, inside the enforcer plugin?  By the way, why is the version of the enforcer plugin upgraded as well?

Reply:
The JDK 11-specific classes were being loaded as part of Jackson's multi-release JAR (META-INF/versions/11/). These classes are intended to be loaded only on JDK 11 and above, but Maven was attempting to load them in my JDK 8 environment, causing compatibility issues.

![image](https://github.com/user-attachments/assets/ce4b2d1b-d563-4dbf-a5d0-ff22ac6e4681)

Reply:
Can you please root cause why we are attempting to load JDK 11 classes and revert this change?

Reply:
The root cause of the issue lies in the multi-release JAR feature in Jackson. The JDK 11-specific classes (located under META-INF/versions/11/) were being loaded even though the build was running on Java 8. This happened because Maven was attempting to load these classes during the build, which caused compatibility issues.

Upgrading the Enforcer plugin to version 3.3.0, along with the extra-enforcer-rules dependency, ensures that Java 8 doesn't attempt to load these JDK 11 classes, as it properly handles multi-release JARs by excluding versioned classes not compatible with the current JDK version.

Reply:
Can you not override the configuration, and use plugin management to update the version?

Reply:
The enforcer plugin and its version upgrade are already managed in the pluginManagement section

Reply:
I have removed the <executions> section, but I suggest keeping the extra-enforcer-rules dependency. It helps resolve compatibility issues with multi-release JARs.This prevents build failures. Would it be okay to keep it for stability?